package.json

```diff
@@ -1,10 +1,10 @@
 {
-   "name":"captchagen",
+   "name":"node-captchagen",
```

Comment:
sorry for this ,this is my first pr in github,maybe make a lot of mistakes. I just want to use it Immediately,so I push a new module to npm. I know you are the owner.And if you republish a new module , I will continue use it and delete mine.i'm so sorry.